### DISCUSSION TITLE
docker run failed

### DISCUSSION THREAD

**Discussion by heyyuyu** [mockoon/mockoon#1000] [Help] [answered]
I am trying to run mockoon-cli through Docker command.
`docker run -d --mount type=bind,source=/opt/mockoon/data/data.json,target=/data,readonly -p 3000:3000 mockoon/cli:latest --data data --port 3000`
It failed to run with the following error:
` ›   Error: This file is not a valid OpenAPI specification (JSON or YAML v2.0.0
 ›    and v3.0.0) or Mockoon environment: EACCES: permission denied, open
 ›   'data'
`
What's the problem?Thank you for your help.

**Comment by 255kb** [comment]
Not sure, but you may need to run Docker with `sudo` if you try to mount from `/opt`.

**Reply by heyyuyu** [reply, marked as answer]
I have figured out the issue.The JSON file does not have executable permission.By the way,Hope to provide cloud-based mockoon services ：）
